modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -608,7 +610,7 @@ private void idleSafeTimeSync() {
                         .groupId(r.join().groupId())
                         .build();
 
-                r.join().processRequest(req);
+                r.join().processRequest(req, clusterNetSvc.topologyService().localMember().id());
```

Comment:
we can calculae the local memeber id once on starting of the manager.

Reply:
fixed